File: Examples/Lesson 6/HotTub.m
//
//  HotTub.m
//  House_main_Swift
//
//  Created by Gabrielle Miller-Messner on 5/9/16.
//  Copyright © 2016 Gabrielle Miller-Messner. All rights reserved.
//

#import "HotTub.h"

@implementation HotTub

-(instancetype)init {
    self = [super init];
    
    _temperature = 102;
    
    return self;
}

-(void)heatUp {
    self.temperature = self.temperature + 1;
}

-(void)coolDown {
    self.temperature = self.temperature - 1;
}

@end

File: Examples/Lesson 6/HotTub.swift
//
//  HotTub.swift
//  House_ObjC_main
//
//  Created by Gabrielle Miller-Messner on 5/9/16.
//  Copyright © 2016 Gabrielle Miller-Messner. All rights reserved.
//

import Foundation

class HotTub: NSObject {
    
    var temperature: Int = 102
    
    func heatUp() {
        self.temperature = temperature + 1
    }
    
    func coolDown() {
        self.temperature = temperature - 1
    }
    
}
